@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1400,31 +1400,49 @@ export class TileGeometryCreator {
         if (tile.projection.type === ProjectionType.Spherical) {
             const { east, west, north, south } = tile.geoBox;
             const sourceProjection = dataSource.getTilingScheme().projection;
-            const g = new THREE.Geometry();
-            g.vertices.push(
-                sourceProjection.projectPoint(new GeoCoordinates(south, west), new THREE.Vector3()),
-                sourceProjection.projectPoint(new GeoCoordinates(south, east), new THREE.Vector3()),
-                sourceProjection.projectPoint(new GeoCoordinates(north, west), new THREE.Vector3()),
-                sourceProjection.projectPoint(new GeoCoordinates(north, east), new THREE.Vector3())
+            const g = new THREE.BufferGeometry();
+            const posAttr = new THREE.BufferAttribute(
+                new Float32Array([
+                    ...sourceProjection
+                        .projectPoint(new GeoCoordinates(south, west), new THREE.Vector3())
```

Comment:
reuse a temporary vector instead of creating new ones

Reply:
Done.